presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -111,6 +115,7 @@
             .put(REAL, WriteMapping.longMapping("real", realWriteFunction()))
             .put(VARBINARY, WriteMapping.sliceMapping("varbinary", varbinaryWriteFunction()))
             .put(DATE, WriteMapping.longMapping("date", dateWriteFunction()))
+            .put(TIME, WriteMapping.longMapping("time", timeWriteFunction()))
```

Comment:
`timeWriteFunction()` is not correct, since TIME (just like TIMESTAMP) has legacy and new semantics, so `timeWriteFunction` needs a session.

Also, `timeWriteFunction()` uses `java.sql.Time` which is not optimal representation when JVM time zone had offset change around 1970-01-01. In particular, the time zone used in tests had such a change.
Instead, it should use `java.time.LocalTime`, similarly to `timestampWriteFunction`. Then, however, it won't be portable between drivers, so it should be added on case-by-case basis (see more below in the other comment).